src/components/addData.js

```diff
@@ -63,10 +63,8 @@ function loadData() {
       window.location.reload();
     });
```

Comment:
Instead of reloading the page on any action of the user, it is better to call a javascript function that loads the updated data without reloading the page.

Reply:
Thanks for that improvement... I did it and works better now! 💯